client/asset/btc/btc.go

```diff
@@ -2013,6 +2001,28 @@ func (btc *ExchangeWallet) send(address string, val uint64, feeRate uint64, subt
 	return nil, 0, 0, fmt.Errorf("failed to locate transaction vout")
 }
 
+func (btc *ExchangeWallet) sendRawTransaction(tx *wire.MsgTx) (*chainhash.Hash, error) {
+	if btc.legacyRawFeeLimit {
+		return btc.node.SendRawTransaction(tx, false)
+	}
+	b, err := serializeMsgTx(tx)
+	if err != nil {
+		return nil, err
+	}
+	var txid string
+	err = btc.node.call(methodSendRawTx,
+		anylist{hex.EncodeToString(b), float64(btc.feeRateLimit) * 1e-5}, &txid)
+	if err != nil {
+		return nil, err
+	}
+	if err != nil {
+		btc.log.Errorf("Error unmarshaling hash returned from sendrawtransaction."+
+			" Expected hash %s: %v", tx.TxHash(), err)
+		txid = tx.TxHash().String()
+	}
```

Comment:
Already returned on error so can't make it to the second condition statement.

Reply:
pffff